qa/L0_metrics/test.sh

```diff
@@ -111,6 +111,23 @@ mkdir -p $MODELDIR/${model}/1 && \
   sed -i "s/label_filename:.*//" config.pbtxt && \
   echo "instance_group [{ kind: KIND_GPU }]" >> config.pbtxt)
 
+### CPU / RAM metrics tests
+set +e
+CLIENT_PY="./cpu_metrics_test.py"
+SERVER_LOG="cpu_metrics_test_server.log"
+# NOTE: CPU utilization is computed based on the metrics interval, so having
+# too small of an interval can skew the results.
+SERVER_ARGS="$BASE_SERVER_ARGS --metrics-interval-ms=1000 --log-verbose=1"
+run_and_check_server
+
+CLIENT_LOG="cpu_metrics_test_client.log"
+python3 ${CLIENT_PY} -v 2>&1 | tee ${CLIENT_LOG}
+check_unit_test
+
+kill $SERVER_PID
```

Comment:
Same here regarding Windows GA. 
```bash
kill $SERVER_PID
wait $SERVER_PID
```
is not Windows friendly. To handle shutdown for our joint Windows/Linux tests, we use:
```bash
kill_server
```
Which is a common function that handles shutdown platform agnostically.